web/src/components/FOI/Home/Redlining.js

```diff
@@ -1935,11 +1934,21 @@ const Redlining = React.forwardRef(
         let incompatableList = incompatibleFiles.filter((doc) =>
           doc.divisions.map((d) => d.divisionid).includes(div.divisionid)
         );
+        let totalPageCount = 0;
+        let totalPagesToRemove =  0;
+        for (let doc of divDocList) {
+          totalPageCount += doc.pagecount;
+          for (let flagInfo of doc.pageFlag) {
+            flagInfo.flagid === pageFlagTypes["Duplicate"] || flagInfo.flagid === pageFlagTypes["Not Responsive"] ? totalPagesToRemove++ : totalPagesToRemove += 0;

```

Comment:
Is it possible to use if condition for this line instead of using "totalPagesToRemove += 0"? Other than that I'm good with the updates. Thanks.